.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+### Expected Behavior
+
+### Actual Behavior
+
+### Steps to Reproduce
+
+### Minimal yet Complete Reproducer Code (or URL to Code)
```

Comment:
not strong here but the intention is that this is a "section header/title" (not a sentence)